public/js/controllers/groupsSection.controller.js

```diff
@@ -10,5 +11,104 @@
         /* jshint validthis: true */
         var vm = this;
 
+        vm.viewGroup = viewGroup;
+        vm.currentGroup = {};
+
+        vm.searchForGroups = searchForGroups;
+        vm.searchString = "";
+        vm.searchResults = [];
+
+        vm.isInGroup = isInGroup;
+        vm.joinGroup = joinGroup;
+        vm.startEditGroup = startEditGroup;
+        vm.leaveGroup = leaveGroup;
+
+        vm.inviteUser = inviteUser;
+        vm.invitationText = "";
+        vm.currentGroupMembers = [];
+        vm.currentGroupNonMembers = [];
+        vm.getGroupMembers = getGroupMembers;
+
+        activate();
+
+        function activate() {
+            vm.searchResults = [
+                {
```

Comment:
Yeah, this data is only for verifying the features while there's no group operations in the data service.